app/models/concerns/fog_extensions/openstack/server.rb

```diff
@@ -1,13 +1,12 @@
 module FogExtensions
   module Openstack
     module Server
-      extend ActiveSupport::Concern
-
-      included do
-        alias_method_chain :security_groups, :no_id
-        attr_reader :nics
-        attr_accessor :boot_from_volume, :size_gb, :scheduler_hint_filter
-        attr_writer :security_group, :network # floating IP
+      def self.prepended(base)
```

Comment:
How about creating a custom `ActiveSupport::Concern` implementation so you can do

```
prepended to

end
```

?

Reply:
you mean something similar to https://gitlab.com/gitlab-org/gitlab-ee/blob/master/config/initializers/0_as_concern.rb ?

Reply:
Yeah. Exactly. But should be a separate PR I think.

Reply:
What's the opinion of other core devs here? Personally I'd rather like to see this PR get in now to speed things up, but no hard feelings either way.

Reply:
I agree. My suggestion should not block this PR at all.